src/versions/develop/pages/BO/catalog/products/index.ts

```diff
@@ -772,7 +773,9 @@ class ProductsPage extends BOBasePage implements BOProductsPageInterface {
       // Move the mouse to avoid the tooltip on first row
       await page.mouse.move(0, 0);
       await this.clickAndWaitForLoadState(page, this.filterResetButton);
-      await this.waitForHiddenSelector(page, this.filterResetButton, 2000);
+      if (!(await this.isClearFilterLinkVisible(page))) {
```

Comment:
```suggestion
      if ((await this.isClearFilterLinkVisible(page))) {
```
Shouldn't it be the opposite logically?